server_test.go

```diff
@@ -93,22 +84,82 @@ func TestServerFromConfig(t *testing.T) {
 	assert.Equal(t, expected, string(body))
 }
 
+func TestPassThroughEnabled(t *testing.T) {
+	targetServer := startTargetServer()
+	defer targetServer.Shutdown(context.TODO())
+
+	viewProxyServer := NewServer("http://localhost:9994")
+	viewProxyServer.Port = 9995
+	viewProxyServer.Logger = log.New(ioutil.Discard, "", log.Ldate|log.Ltime)
+	viewProxyServer.PassThrough = true
+
+	go func() {
```

Comment:
This starts a go-routtine that will be killed once this function exits?

Reply:
That was the intention, but it looks like I forgot to close it, so it will live forever!

I added `defer viewProxyServer.Close()` in a commit I just pushed, so it will be closed once the test is finished.